helix-term/src/commands.rs

```diff
@@ -471,7 +471,9 @@ impl MappableCommand {
         decrement, "Decrement item under cursor",
         record_macro, "Record macro",
         replay_macro, "Replay macro",
-        command_palette, "Open command palette",
+        command_palette, "Open command pallete",
```

Comment:
Looks like "palette" was swapped out for the incorrect "pallete" at some point?

Reply:
Resolved by https://github.com/pinelang/helix-tree-explorer/commit/f5aec54fe2b20305a9288f6fba0bd376d3fa491b